game_eggs/steamcmd_servers/core_keeper/egg-core-keeper.json

```diff
@@ -0,0 +1,162 @@
+            "field_type": "text"
+        },
+        {
+            "name": "Game ID",
+            "description": "Game ID to use for the server. Needs to be at least 28 characters and alphanumeric, excluding Y,y,x,0,O. Empty or not valid means a new ID will be generated at start. Default: EMPTY.",
+            "env_variable": "GAME_ID",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string|min:28",
+            "field_type": "text"
+        },
+        {
+            "name": "Data Path",
+            "description": "Save file location. Defaul: DedicatedServer.",
+            "env_variable": "DATA_PATH",
+            "default_value": "DedicatedServer",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:100",
+            "field_type": "text"
+        },
+        {
+            "name": "Max Players",
+            "description": "Maximum number of players that will be allowed to connect to server. Maximum is 100. Default: 100,",
+            "env_variable": "MAX_PLAYERS",
+            "default_value": "100",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|min:1|max:100",
+            "field_type": "text"
+        },
+        {
+            "name": "World Mode",
+            "description": "Whether to use normal (0) or hard (1) mode for world. Default: 0.",
+            "env_variable": "WORLD_MODE",
+            "default_value": "0",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer|min:0|max:1",
+            "field_type": "text"
+        },
+        {
+            "name": "Steam App ID",
+            "description": "Steam App ID of Core Keeper Dedicated Server. Found here: https:\/\/steamdb.info\/search\/?a=app&q=core+keeper&type=5&category=0",
+            "env_variable": "SRCDS_APPID",
+            "default_value": "1963720",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|integer|min:1",
+            "field_type": "text"
+        },
+        {
+            "name": "Auto Update",
+            "description": "Enable automatic updates on boot: Default: true",
+            "env_variable": "AUTO_UPDATE",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|boolean",
+            "field_type": "text"
+        },
+        {
+            "name": "XVFB",
+            "description": "Enable virtual framebuffer. Fix: 1",
+            "env_variable": "XVFB",
+            "default_value": "1",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|integer",
+            "field_type": "text"
+        },
+        {
+            "name": "DISPLAY_WIDTH",
```

Comment:
not needed , set by default in the docker image

Reply:
Yes, that was my thought too. Unfortunately, Xvfb has to be started with with a color depth of 24, otherwise there will be an error when starting the Core Keeper server. Default in entrypoint.sh is 1024x768x16.

Error with 16bit color depth (1024x769x16 or 1x1x16):
```
Desktop is 1024 x 768 @ 0 Hz
X Error of failed request:  BadMatch (invalid parameter attributes)
  Major opcode of failed request:  1 (X_CreateWindow)
```

```
Desktop is 1 x 1 @ 0 Hz
X Error of failed request:  BadMatch (invalid parameter attributes)
  Major opcode of failed request:  1 (X_CreateWindow)
```

I have adjusted all three variables so that they correspond to the original values ​​in the supplied start script: 1x1x24.

Reply:
Fine for me

Reply:
> not needed , set by default in the docker image

This do not really work the ENV from dockerfile